sdk/ai/azure-ai-client/azure/ai/client/models/_patch.py

```diff
@@ -91,6 +91,28 @@ def __init__(self, *, connection: ConnectionsListSecretsResponse, token_credenti
                 self.key = connection.properties.credentials.key
         self.token_credential = token_credential
 
+    def to_evaluator_model_config(self, deployment_name, api_version) -> Dict[str, str]:
+        connection_type = self.connection_type.value
+        if self.connection_type.value == "AzureOpenAI":
```

Comment:
`== ConnectionType.AZURE_OPEN_AI` instead of `== "AzureOpenAI"`

Reply:
done